### PR TITLE
Add OMIM data source

### PR DESCRIPTION
**Pull request by haochenl**

### CODE REVIEW

File: docs/data-sources/omim.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+title: OMIM
+---
+
+import JSON from './omim-json.md'
+
+## Overview
+
```
Comment (rajatshuvro) [inline docs/data-sources/omim.mdx:8]: In the overview , I usually like to provide a short introduction to the data source. Details of parsing and using the data source should be in a subsequent section in my opinion.

File: docs/data-sources/omim.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+100600  phenotype
+100640  gene    216 ALDH1A1 ENSG00000165092
+100650  gene/phenotype  217 ALDH2   ENSG00000111275
+100660  gene    218 ALDH3A1 ENSG00000108602
+100670  gene    219 ALDH1B1 ENSG00000137124
+100675  predominantly phenotypes
+100678  gene    39  ACAT2   ENSG00000120437
+```
+
+The information in the "Entrez Gene ID (NCBI)",  "Approved Gene Symbol (HGNC)" and "Ensembl Gene ID (Ensembl)" columns are used to find the proper gene symbol supported by Nirvana, which may or may not be the same as the gene symbol listed here.  Every MIM number that has a Nirvana supported gene symbol will be put into the mapping file.download
```
Comment (rajatshuvro) [inline docs/data-sources/omim.mdx]: is there a .download file or is this a typo?

File: docs/data-sources/omim.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+OMIM API can accept almost 20 MIM numbers in one query. To simplify the work to parsing the JSON responses, this gzipped json response file has all the original responses properly concatenated as if they were returned by the server in a single query.
+
+### 3. MimEntries.json.gz.version
+
+The version file is generated automatically after all the data have been downloaded. The date in the local time zone is used to version the data. This means, if the downloader has been run twice a day, the version of downloaded data would be the same, although OMIM website may have an update between the two runs.
+
+## Generate OMIM database
+
+```shell
```
Comment (rajatshuvro) [inline docs/data-sources/omim.mdx]: I am all for describing the data parsing and filtering process but we should not provide SAUtils commands in this documentation I think.

**Review by rajatshuvro — CHANGES_REQUESTED** [review]
Please see specific comments.

File: docs/data-sources/omim.mdx (hunk 1 above)
Comment (haochenl) [reply docs/data-sources/omim.mdx:8]: I added a description of OMIM

File: docs/data-sources/omim.mdx (hunk 2 above)
Comment (haochenl) [reply docs/data-sources/omim.mdx]: It is a typo I think.

File: docs/data-sources/omim.mdx (hunk 3 above)
Comment (haochenl) [reply docs/data-sources/omim.mdx]: We can talk about this.

**Pull request merged**